static/css/inline-selection.scss

```diff
@@ -402,6 +402,20 @@ $very-light-grey: #ededed;
   width: 400px;
 }
 
+#imageCroppedWarning {
+  background: rgba(0,0,0,.8);
```

Comment:
Could you specify the opacity separately? Seems to make Windows High Contrast mode behave in a more reasonable way

Reply:
I don't think you can make the background opacity separate, only the entire element opacity...?